app.py

```diff
@@ -30,11 +29,67 @@ def send_mail(form_msg, form_name, form_email):
         # print("sent")
     except urllib.HTTPError:
```

Comment:
This must be urllib.error.HTTPError and you have to import urllib module at the beginning. I tested this.